### PR TITLE
Fixed #329

### LINKED ISSUE #329: Disabled button inside a MaterialDesignMultiFloatingActionAccentPopupBox
When a button inside a `MaterialDesignMultiFloatingActionAccentPopupBox` is disabled, make it grayish to make it looked disabled.

### PR DESCRIPTION
**Pull request by CarHel**
I just removed the lines on the popupbox setting Opacity.

### CODE REVIEW

File: MaterialDesignThemes.Wpf/PopupBox.cs [hunk 1]
```diff
@@ -579,12 +579,6 @@ private void AnimateChildrenIn(bool reverse)
                 uiElement.SetCurrentValue(RenderTransformOriginProperty, new Point(.5, .5));
                 uiElement.RenderTransform = transformGroup;
 
-                var opacityAnimation = new DoubleAnimationUsingKeyFrames();                
-                opacityAnimation.KeyFrames.Add(new EasingDoubleKeyFrame(0, absoluteZeroKeyTime, sineEase));
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/PopupBox.cs]: This is removing the subtle fade in effect.  I think we need something like this here:
```
if (uiElement.IsEnabled)
{
```

File: MaterialDesignThemes.Wpf/PopupBox.cs [hunk 2]
```diff
@@ -609,7 +603,6 @@ private void AnimateChildrenIn(bool reverse)
                 Storyboard.SetTarget(translateCoordinateAnimation, uiElement);
 
                 var storyboard = new Storyboard();
-                storyboard.Children.Add(opacityAnimation);
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/PopupBox.cs]: if (uiElement.IsEnabled)

**Comment by CarHel** [comment]
opacityAnimation.KeyFrames.Add(new EasingDoubleKeyFrame((double)uiElement.GetAnimationBaseValue(OpacityProperty), deferredEndKeyTime, sineEase));  
That is the line that i changed

**Comment by ButchersBoy** [comment]
Excellent, thank you, and welcome to the contributors list!

**Pull request merged**